File: public/docs/_examples/toh-5/ts/app/app.component.1.ts
/* tslint:disable no-unused-variables */
// #docplaster
// #docregion
import { Component }       from '@angular/core';

import { HeroService }     from './hero.service';
import { HeroesComponent } from './heroes.component';
// #enddocregion

// For testing only
import { ROUTER_DIRECTIVES } from '@angular/router';

// #docregion
@Component({
  selector: 'my-app',
  template: `
    <h1>{{title}}</h1>
    <my-heroes></my-heroes>
  `,
  directives: [HeroesComponent],
  providers: [
    // #enddocregion
    // #docregion
    HeroService
  ]
})
export class AppComponent {
  title = 'Tour of Heroes';
}

File: public/docs/_examples/toh-5/dart/lib/app_component_1.dart
// #docplaster
// #docregion
import 'package:angular2/core.dart';
// #enddocregion
import 'package:angular2/router.dart'; // for testing only
// #docregion

import 'hero_service.dart';
import 'heroes_component.dart';

@Component(
    selector: 'my-app',
    template: '''
      <h1>{{title}}</h1>
      <my-heroes></my-heroes>''',
    directives: const [HeroesComponent],
    providers: const [
      // #enddocregion
      ROUTER_PROVIDERS,
      // #docregion
      HeroService
    ])
class AppComponent {
  String title = 'Tour of Heroes';
}
